Failed to compile in openSUSE 42.2

I tried to compile iTALC 3.0.3 on openSUSE 42.2, and I got the following error:

`
_### _[ 28%] Building CXX object lib/CMakeFiles/ItalcCore.dir/src/Ipc/QtSlaveLauncher.o
[ 28%] Building CXX object lib/CMakeFiles/ItalcCore.dir/src/Ipc/Slave.o
[ 28%] Building CXX object lib/CMakeFiles/ItalcCore.dir/src/Ipc/SlaveLauncher.o
/home/user/bin/Source_code_installers/iTALC/italc/lib/src/Ipc/QtSlaveLauncher.cpp: In member function ‘virtual void Ipc::QtSlaveLauncher::start(const QStringList&)’:
/home/user/bin/Source_code_installers/iTALC/italc/lib/src/Ipc/QtSlaveLauncher.cpp:70:42: error: no matching function for call to ‘Ipc::QtSlaveLauncher::connect(QPointer<QProcess>&, void (QProcess::*)(int, QProcess::ExitStatus), QPointer<QProcess>&, void (QObject::*)())’
        m_process, &QProcess::deleteLater );
                                          ^
/home/user/bin/Source_code_installers/iTALC/italc/lib/src/Ipc/QtSlaveLauncher.cpp:70:42: note: candidates are:
In file included from /usr/include/qt5/QtCore/qcoreapplication.h:40:0,
                 from /usr/include/qt5/QtCore/QCoreApplication:1,
                 from /home/user/bin/Source_code_installers/iTALC/italc/lib/src/Ipc/QtSlaveLauncher.cpp:27:
/usr/include/qt5/QtCore/qobject.h:196:36: note: static QMetaObject::Connection QObject::connect(const QObject*, const char*, const QObject*, const char*, Qt::ConnectionType)
     static QMetaObject::Connection connect(const QObject *sender, const char *signal,


lib/CMakeFiles/ItalcCore.dir/build.make:729: recipe for target 'lib/CMakeFiles/ItalcCore.dir/src/Ipc/QtSlaveLauncher.o' failed
make[2]: *** [lib/CMakeFiles/ItalcCore.dir/src/Ipc/QtSlaveLauncher.o] Error 1
make[2]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:2309: recipe for target 'lib/CMakeFiles/ItalcCore.dir/all' failed
make[1]: *** [lib/CMakeFiles/ItalcCore.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2
user@linux-home-pc:~/bin/Source_code_installers/iTALC/italc>__ 
`

--- a/lib/src/Ipc/QtSlaveLauncher.cpp
+++ b/lib/src/Ipc/QtSlaveLauncher.cpp
@@ -65,7 +65,7 @@ void QtSlaveLauncher::start( const QStringList &arguments )
 	// forward stdout/stderr from slave to master
 	m_process->setProcessChannelMode( QProcess::ForwardedChannels );
 
-#if QT_VERSION >= 0x050300
+#if QT_VERSION >= 0x050700
 	QObject::connect( m_process, static_cast<void(QProcess::*)(int, QProcess::ExitStatus)>(&QProcess::finished),
 					  m_process, &QProcess::deleteLater );
 	QObject::connect( m_process, static_cast<void(QProcess::*)(int, QProcess::ExitStatus)>(&QProcess::finished),

